content/applications/general/email_communication/faq.rst

```diff
@@ -1,174 +1,185 @@
-===
-FAQ
-===
+=========
+Email FAQ
+=========
 
-This document contains an explanation of the most recurring mailing concerns.
-
-We will start by addressing issues of outgoing emails (ex: my client has not received my email),
-and then, of incoming emails (ex: I do not receive responses from my customers in the database).
+This document contains an explanation of the most recurring emailing issues in Odoo. Outgoing emails
+(ex: the client has not received the email from the database) will be addressed first, and then,
+incoming emails (ex: Responses have not been received from customers in the database).
 
 Outgoing emails
 ===============
 
 .. _red_envelop:
 
-What do you have to check if your email is not sent?
-----------------------------------------------------
+What to check if the email is not sent?
+---------------------------------------
 
-The first indicator showing you that the email has not been sent is the red envelope next to the
-date and time of the message.
+The first indicator showing that the email has not been sent is the red envelope next to the date
+and time of the message in the chatter.
 
 .. image:: faq/red-envelop.png
    :align: center
-   :alt: Red envelope displayed in chatter
+   :alt: Red envelope displayed in chatter.
+
+Unsent emails will also appear in the Odoo email queue. In :ref:`Developer mode <developer-mode>`
+this can be accessed by going to :menuselection:`Settings --> Technical Menu --> Emails --> Emails`.
+Unsent emails will appear in turquoise; sent emails will appear in grey.
 
 Common error messages
 ~~~~~~~~~~~~~~~~~~~~~
 
 .. _email_communication/daily_limit_mail:
 
-You reached your daily limit:
-*****************************
+Reached the daily limit:
+************************
 
 .. image:: faq/email-limit.png
    :align: center
-   :alt: Warning in Odoo upon email limit reached
+   :alt: Warning in Odoo upon email limit reached.
 
 Each email service provider has its own email sending limits. The limits may be daily, hourly,
-and sometimes even per minute. This is the same for Odoo, we have to limit our customers to prevent
-our e-mail servers from being blacklisted.
+and sometimes even per minute. This is the same for Odoo, which limits a customer's sending to
+prevent Odoo's e-mail servers from being blacklisted.
 
 Here are the default limits for new databases:
 
 - 200 emails/day for Odoo Online and Odoo.sh databases with an active subscription,
 
 - 50 emails/day for one-app free and trial databases,
```

Comment:
```suggestion
- 200 emails/day for Odoo Online and Odoo.sh databases with an active subscription.

- 50 emails/day for one-app free and trial databases.
```
replaced commas with periods - unsure if the commas were necessary in this list-like format.